### PR TITLE
Quick fix for 3w regions blank page

### PR DESCRIPTION
**Pull request by szabozoltan69**
@batpad @frozenhelium @tnagorra – if you have better solution, please tell me.
Refers to https://github.com/IFRCGo/go-frontend/issues/2555

### CODE REVIEW

**Comment by frozenhelium** [comment]
@szabozoltan69 The right approach would be to add a `/` at the end of the regions link in the 3W page. This approach reveals another mistake in the Multiplexer. We've set the route of Regions to match with `exact` which is not desired behaviour (Since, we have the hash). I've fixed it now, please do review :)

**Review by frozenhelium — APPROVED** [review]

**Review by frozenhelium — APPROVED** [review]

File: src/root/views/region.js [hunk 1]
```diff
@@ -568,7 +568,7 @@ const selector = (state, ownProps) => ({
   appealsListStats: state.overallStats.appealsListStats,
   countriesByRegion: countriesByRegionSelector(state),
   regions: regionsByIdSelector(state),
-  thisRegion: regionByIdOrNameSelector(state, ownProps.match.params.id),
+  thisRegion: regionByIdOrNameSelector(state, ownProps.match.params.id.split('#')[0]),
```
Comment (szabozoltan69) [inline src/root/views/region.js:571]: @frozenhelium should I remove this cutdown also?

**Comment by frozenhelium** [comment]
@szabozoltan69 turns out it was the problem with the link fix, I've pushed with the appropriate fix now

File: src/root/views/region.js (hunk 1 above)
Comment (frozenhelium) [reply src/root/views/region.js]: I did it in my latest push :D

**Review by frozenhelium — APPROVED** [review]

**Comment by szabozoltan69** [comment]
Beautiful and clean solution, thank you so much, @frozenhelium!

**Comment by szabozoltan69** [comment]
Deployed to Staging.

**Pull request merged**